### PR TITLE
added asciidoctor.asciidctor-vscode

### PR DESCRIPTION
**Pull request by andhet**
Enhanced extensions.json with the AsciiDoc-Plugin released under the MIT license. (see https://marketplace.visualstudio.com/items?itemName=asciidoctor.asciidoctor-vscode)

### CODE REVIEW

**Review by spoenemann — COMMENTED** [review]
Thanks for the contribution!

The extension has version `0.0.0` in its package.json:
https://github.com/asciidoctor/asciidoctor-vscode/blob/24f5a90a08e6b8931257e1f619bee33e0d196cd9/package.json#L5

We could solve this by downloading the already packaged vsix file instead of building it from sources:

```
    {
      "id": "asciidoctor.asciidoctor-vscode",
      "version": "2.8.4",
      "download:" "https://github.com/asciidoctor/asciidoctor-vscode/releases/download/v2.8.4/asciidoctor-vscode-2.8.4.vsix"
    },
```

File: extensions.json [hunk 1]
```diff
@@ -94,6 +94,12 @@
       "id": "asabil.meson",
       "repository": "https://github.com/asabil/vscode-meson"
     },
+    {
+      "id": "asciidoctor.asciidctor-vscode",
```
Comment (spoenemann) [inline extensions.json:98]: Typo: `asciidoctor-vscode`

Comment (andhet) [reply extensions.json]: Commited requested changes. Hopefully this time without introducing a new typo...

File: extensions.json [hunk 2]
```diff
@@ -94,6 +94,12 @@
       "id": "asabil.meson",
       "repository": "https://github.com/asabil/vscode-meson"
     },
+    {
+      "id": "asciidoctor.asciidoctor-vscode",
+      "repository": "https://github.com/asciidoctor/asciidoctor-vscode",
```
Comment (spoenemann) [inline extensions.json:99]: Please remove the `repository` property. It is not necessary when we have a download URL.

**Review by spoenemann — COMMENTED** [review]
Great, just one thing left:

File: extensions.json (hunk 2 above)
Comment (andhet) [reply extensions.json]: Sorry. Commit done.

**Review by spoenemann — APPROVED** [review]
:+1:

**Pull request merged**